docs/source/analyze/cluster-resource.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Cluster Resource
+description: Analyze any attribute collected by the `ClusterResources` Collector
+---
+
+The `clusterResource` analyzer can be used to check any attribute of any known resources in the cluster in a generic manner.
+
+The `clusterResource` analyzer uses data from the [clusterResources collector](/collect/cluster-resources).
+The `clusterResources` collector is automatically added and will always be present.
+
+You must specify the `kind` and `name` attributes, plus the optional `namespace` attribute to target the Kubernetes resource.
+
+The `yamlPath` attribute is used to specify a dot-delimited YAML path of a property on the Kubernetes resource referenced in `name`.
+The `when` attribute supports standard arithmetic comparison operators.
+
+The outcomes on this analyzer will be processed in order, and execution will stop after the first outcome that is truthy.
+
+## Parameters
+
+**checkName**: (Optional) Analyzer name.
+Used for uniqueness if multiple analyzers are defined with similar parameters.
+
+**kind**: (Required) The type of Kubernetes resource being targeted by `name`.
+
+**name**: (Required) The name of the resource to check.
+
+**namespace**: (Optional) The namespace to look for the resource in.
```

Comment:
it's worth noting here that the absence of `namespace` configures the analyzer to search for cluster-scoped resources

Reply:
done thx